rules/macos/process_creation/proc_creation_macos_enable_guest_account.yml

```
@@ -0,0 +1,23 @@
+title: Enable the Guest Account
+id: d7329412-13bd-44ba-a072-3387f804a106
+status: experimental
+description: Detects attempts to enable the guest account using the sysadminctl utility
+references:
+    - https://ss64.com/osx/sysadminctl.html
+author: Sohan G (D4rkCiph3r)
+date: 2023/02/18
+tags:
+    - attack.t1078
+    - attack.t1078.001
+    - attack.initial_access
+logsource:
+    category: process_creation
+    product: macos
+detection:
+    selection:
+        Image|endswith: '/sysadminctl'
+        CommandLine|contains: 'guestAccount on' #by default the guest account is not active
```

```suggestion
        CommandLine|contains: '-guestAccount on' #by default the guest account is not active
```
With the "-" , see it is a cli option when read the rule